src/app/domain/geolocation/tracking/tracking.js

```diff
@@ -247,14 +271,21 @@ const addPoint = (content, point, filtered) => {
   }
 }
 
-const addMotionActivity = (content, previousPoint, point) => {
-  if (!previousPoint || previousPoint.activity?.type !== point.activity?.type) {
-    // Add new activity type when it's the first point, or when a new motion activity type (i.e. mode) is detected
-    // TODO: might be more relevant to use the activity events
-    const motionActivity = translateToEMissionMotionActivityPoint(point)
-    content.push(motionActivity)
+const getActivityMode = activity => {
+  const modeKeys = [
+    'cycling',
+    'running',
+    'walking',
+    'automotive',
+    'stationary',
+    'unknown'
+  ]
```

Comment:
You can move this constant outside the function.

Reply:
Fair enough, I move it